filament/include/filament/Texture.h

```diff
@@ -236,6 +236,14 @@ class UTILS_PUBLIC Texture : public FilamentAPI {
          */
         Builder& import(intptr_t id) noexcept;
 
+        /**
+         * Specifies the sample count of the texture
+         *
+         * @param sampleCount sample count
```

Comment:
nit: a comment about default would be nice here too
```suggestion
         * @param sampleCount sample count (default: 1)
```

Reply:
fixed in https://github.com/shapr3d/filament/pull/67/commits/917a91248e3bd5fdf063c55e3dcdbf473fc76d29